atl-checker/src/main.rs

```diff
@@ -160,6 +159,42 @@ fn main_inner() -> Result<(), String> {
                 },
             )?
         }
+        ("analyse", Some(analyse_args)) => {
+            let input_model_path = analyse_args.value_of("input_model").unwrap();
+            let model_type = get_model_type_from_args(&analyse_args)?;
+            let formula_path = analyse_args.value_of("formula").unwrap();
+            let formula_format = get_formula_format_from_args(&analyse_args)?;
+
+            fn analyse_model<G>(graph: ATLDependencyGraph<G>, v0: ATLVertex)
+            where
+                G: GameStructure + Send + Sync + Clone + Debug + 'static,
+            {
+                println!("Analysing EDG ...");
```

Comment:
```suggestion
                println!("Analysing EDG (state formula: length)");
```

Interpreting output is a bit difficult, when you don't know what you're looking at. I think we should include a small hint like this. Maybe length should be depth instead?